### PR TITLE
refactor: new pbr interface

### PR DESCRIPTION
**Pull request by zhuxudong**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Feature.
### What is the current behavior? (You can also link to an open issue here)
PBR interface refactor. #195 
### What is the new behavior (if this is a feature change)?
Refactored.
### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
Yes.

* delete `opacity`
* delete `opacityTexture`
* delete `reflectionTexture`
* delete `envMapIntensity`
* delete `refractionRatio`
* delete `refractionDepth`
* delete `refractionTexture`
* delete `perturbationTexture`
* delete `perturbationUOffset`
* delete `perturbationVOffset`
* delete `srgb`
* delete `srgbFast`
* delete `gamma`
* delete `getOpacityFromRGB`
* delete `envMapModeRefract`
* delete `metallicTexture`
* delete `roughnessTexture`
* `metallicFactor` *> `metallic`
* `roughnessFactor` *> `roughness`
* `metallicRoughnessTexture` *> `roughnessMetallicTexture`
* `glossinessFactor` *> `glossiness`
* `normalIntensity` *> `normalTextureIntensity`
* `occlusionStrength` *> `occlusionTextureIntensity`

### Other information:

### CODE REVIEW

File: packages/core/src/material/PBRBaseMaterial.ts [hunk 1]
```diff
@@ -403,24 +159,17 @@ export abstract class PBRBaseMaterial extends BaseMaterial {
    */
   constructor(engine: Engine) {
```
Comment (yangfengzzz) [inline packages/core/src/material/PBRBaseMaterial.ts:160]: constructor in abstract class can be protected

File: packages/core/src/material/PBRBaseMaterial.ts [hunk 2]
```diff
@@ -403,24 +159,17 @@ export abstract class PBRBaseMaterial extends BaseMaterial {
...
     super(engine, Shader.find("pbr"));
```
Comment (yangfengzzz) [inline packages/core/src/material/PBRBaseMaterial.ts:161]: constructor in abstract class can be protected

File: packages/loader/src/scene-loader/resources/PBRSpecularMaterialResource.ts [hunk 3]
```diff
@@ -6,30 +6,21 @@ import { TextureResource } from "./TextureResource";
 
 const attrs = [
```
Comment (yangfengzzz) [inline packages/loader/src/scene-loader/resources/PBRSpecularMaterialResource.ts:7]: 'getAllGetters' is not used in this file, which can be deleted in the import.

**Review by GuoLei1990 — APPROVED** [review]
+1

**Pull request merged**